assets/js/modules/adsense/util/index.js

```diff
@@ -21,12 +21,16 @@
  */
 import each from 'lodash/each';
 
+/*
+ * Internal dependencies
+ */
 export * from './is-zero-report';
 export * from './parsing';
 export * from './site-stats-data';
 export * from './status';
 export * from './validation';
 export * from './url';
+import { stringToDate } from '../../../util/date-range/string-to-date';
```

Comment:
@sashadoes Please move the import directly under Internal dependencies. Exports should not be under internal dependencies.